ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/messages/MessageWrapperTest.java

```diff
@@ -231,6 +235,62 @@ public void PooledTransactions() throws IOException {
     assertThat(actual).isEqualTo(expected);
   }
 
+  @Test
+  public void readFromExpectsListWrappingBodyFields() {
+    // BodiesMessages broadcast bodies in this format
+    // [[txs],[ommers],[withdrawals]]
+    // 0xc3c0c0c0
+    final BytesValueRLPInput bytesValueRLPInput =
+        new BytesValueRLPInput(Bytes.fromHexString("0xc3c0c0c0"), false);
+    final BlockBody blockBodyDecodefromRLP =
+        BlockBody.readWrappedBodyFrom(bytesValueRLPInput, new MainnetBlockHeaderFunctions());
+
+    assertThat(blockBodyDecodefromRLP)
+        .isEqualTo(
+            new BlockBody(
+                Collections.emptyList(),
+                Collections.emptyList(),
+                Optional.of(Collections.emptyList()),
+                Optional.empty()));
+  }
+
+  @Test
+  public void readBodyFieldsExpectsNoListWrappingBodyFields() {
+    // rlps of blocks contains block data in this format
```

Comment:
What does "rlps of blocks" mean? Isn't BodiesMessages rlps of blocks as well?

Reply:
think the input when using the block import feature is an rlp of blocks. BodiesMessages would be a rlp of Bodies only is my understand.